pkg/asset/tls.go

```diff
@@ -98,13 +109,13 @@ func newAPIKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey, altNa
 	return key, cert, err
 }
 
-func newKubeletKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey) (*rsa.PrivateKey, *x509.Certificate, error) {
+func newAdminKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey) (*rsa.PrivateKey, *x509.Certificate, error) {
 	key, err := tlsutil.NewPrivateKey()
 	if err != nil {
 		return nil, nil, err
 	}
 	config := tlsutil.CertConfig{
-		CommonName:   "kubelet",
+		CommonName:   "admin",
 		Organization: []string{"kube-node"},
```

Comment:
Is `kube-node` required? Otherwise we could drop it.

Also, right above is the cert generation for the API Server. It has the `kube-master` group. I feel like it should be renamed too.